integr8ly/integreatly-operator
# manually update production manifest for rhoam

# Issue link
No jira - manual generation of manifest for rhoam due to issue - https://issues.redhat.com/browse/MGDAPI-2728

# What
Updated manifest for RHOAM

# Verification steps
Verify that only manifest is updated and nothing else
Verify that version is correct
